README.md

```diff
@@ -42,6 +42,8 @@ Other endpoints:
 * ``/checks/{a-project}``: execute all checks of project ``a-project``
 * ``/checks/tags/{a-tag}``: execute all checks with tag ``a-tag``
 
+Cache can be forced to be bypassed with ``?refresh={s3cr3t}``. See *Environment variables* section.
```

Comment:
This makes it sound like the parameter works for the "other endpoints" above, while the code looks like it only works for individual checks (not for projects or tags). Could we clarify this?